dashboard/.env.example

```diff
@@ -13,4 +13,8 @@ POSTGRES_HOST=database
 
 # PG Admin config
 PGADMIN_DEFAULT_EMAIL=admin@example.com
-PGADMIN_DEFAULT_PASSWORD=adminpassword
\ No newline at end of file
+PGADMIN_DEFAULT_PASSWORD=adminpassword
+
+# AWS config
+AWS_ACCESS_KEY_ID = somekey
```

Comment:
Upar space nhi hai `=` ke beech me to idhar kyu hai?

Reply:
True